src/targets/gpu/include/migraphx/gpu/rocblas.hpp

```diff
@@ -25,6 +25,9 @@
 #define MIGRAPHX_GUARD_MIGRAPHLIB_ROCBLAS_HPP
 #include <migraphx/manage_ptr.hpp>
 #include <migraphx/config.hpp>
+// for development:
+#define ROCBLAS_BETA_FEATURES_API
```

Comment:
We can check if BETA API exists  in  installed rocBLAS or not inside CMake and define ROCBLAS_BETA_FEATURES_API from there. Similar to this : 
https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/a63ee2e0a81aa3a59ff65d7aa67b522993977274/src/targets/gpu/CMakeLists.txt#L221

Reply:
Since it is BETA api, all the logic related to tuning should be guarded based on whether #ROCBLAS_BETA_FEATURES_API is defined or not.

Reply:
done

Reply:
I assume this variable will have to be renamed some day when rocBLAS commits these features to production.